### PR TITLE
Remove extraneous ) in SQL, fix bug #28008

### PR DESCRIPTION
**Pull request by j16sdiz**
http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=28008

### CODE REVIEW

File: libraries/joomla/table/session.php [hunk 1]
```diff
@@ -132,7 +132,7 @@ public function purge($maxLifetime = 1440)
 		$query = $this->_db->getQuery(true);
 		$query->delete();
 		$query->from($this->_db->quoteName($this->_tbl));
-		$query->where($this->_db->quoteName('time') . ' < \'' . (int) $past . '\')');
+		$query->where($this->_db->quoteName('time') . ' < ' . $this->quote($past));
```
Comment (chdemko) [inline libraries/joomla/table/session.php:135]: sorry for the mistake, it should be

```
$this->_db->quote($past)
```

**Comment by elinw** [comment]
I think when it is an (int) you don't use quote.

**Comment by chdemko** [comment]
@elinw +1

**Comment by chdemko** [comment]
Fixed in code

**Pull request closed without merging**